components/footer/footer.scss

```diff
@@ -22,45 +22,58 @@
       justify-content: space-between;
     }
 
-    p {
-      margin-top: 0;
-      margin-bottom: $base;
-    }
-
     &-col {
       display: flex;
       flex-direction: column;
-      color: $secondary;
       line-height: 20px;
 
       .links {
-        margin-bottom: $base;
+        display: flex;
+
+        @include xsm {
+          flex-direction: column;
+        }
+
+        @include sm {
+          flex-direction: row;
+          justify-content: space-between;
+        }
 
         a {
           @include link-underline;
```

Comment:
if you do `@include link-underline(white)` it'll change the :visited colour too, so no need to have `color: white`